perllib/FixMyStreet/App/Form/Page/Waste.pm

```diff
@@ -6,11 +6,21 @@ extends 'FixMyStreet::App::Form::Page::Simple';
 has title => ( is => 'ro', 'isa' => 'Str', lazy => 1, builder => '_build_title' );
 # So we can insert cobrand specific service name
 has title_ggw => ( is => 'ro', isa => 'Str' );
+has bulky_general_title => ( is => 'ro', 'isa' => 'Str' );
 
 sub _build_title {
     my $self = shift;
-    my $name = $self->form->{c}->cobrand->garden_service_name;
-    return sprintf($self->title_ggw, $name);
+    if ($self->form =~ /Bulky/) {
```

Comment:
Perhaps could subclass Form/Page/Waste.pm and override _build_title there? `default_page_type` on Form/Waste/Bulky.pm selects which "Page" class to use.